src/goto-programs/goto_convert.cpp

```diff
@@ -419,14 +419,8 @@ void goto_convertt::convert_block(const codet &code, goto_programt &dest)
 
 void goto_convertt::convert_expression(const codet &code, goto_programt &dest)
 {
-  if(code.operands().size() != 1)
-  {
-    err_location(code);
-    message_handler.error("expression statement takes one operand\n");
-    abort();
-  }
-
-  exprt expr = code.op0();
+  // Old frontend may have the target operand at op1 rather than op0.
+  exprt expr = (code.operands().size() != 1) ? code.op1() : code.op0();
```

Comment:
can we change the old frontend to set the target in op0?

I'm not always in favor of fixing the old frontend but I don't think we should change the rest of esbmc because of it, as it might introduce bugs that we simply don't need to fix.

Reply:
Agreed. This might introduce some unexpected bugs in the future. I'll fix the old frontend instead of goto converter.

Reply:
yeah, or you can simply ignore it! I mean, the purpose of re-enabling the old frontend is to use the symbol table as a reference, right? If so, you shouldn't worry about the verification being broken later in the verification pipeline.